ModelTraining/finetuning.py

```diff
@@ -0,0 +1,283 @@
+    shutil.rmtree(dataPath/"new_data/")
+
+pre_process(Path("./data/"))
+
+"""
+Resulting Folder Structure - 
+./data/   
+    |-positive/
+        - xyz1.wav
+        - xyz2.wav
+        ...
+    |-negative/
+        - abc1.wav
+        - abc2.wav
+        ..
+    |-new_samples/
+        - asdfas.wav
+        - asdfas.wav
+        ..
+"""
+
+def data_blender(dataPath):
+    """
+    Function to blend the original data with new data
+    
+    """
+    pos_samples  = len((dataPath/'positive').ls())
+    neg_samples  = len((dataPath/'negative').ls())
+    new_neg_samples  = len((dataPath/'new_samples').ls())
+    total_neg_samples = neg_samples + new_neg_samples
+    
+    neg_img_list = pd.Series((dataPath/'negative').ls() + (dataPath/'new_samples').ls())
+    
+    ## Randomly selecting neg_samples for overall list
+    new_neg_img_list = neg_img_list.sample(n=neg_samples, replace=False).values
+    
+    ## Get image name
+    new_neg_img_list = [str(item).split('/')[-1] for item in new_neg_img_list]
+    
+    ## copying all data from new samples to negative
+    for item in (dataPath/'new_samples').ls():
+        shutil.move(src= str(item), dst = dataPath/'negative', copy_function = shutil.copy)
+        
+    ## removing new samples directory
+    shutil.rmtree(dataPath/'new_samples')
+        
+    for item in (dataPath/'negative').ls():
+        if str(item).split('/')[-1] not in new_neg_img_list:
+            os.remove(item)
+
+data_blender()
+"""
+Resulting Folder Structure - 
+./data/   
+    |-positive/
+        - xyz1.wav
+        - xyz2.wav
+        ...
+    |-negative/
+        - abc1.wav
+        - abc2.wav
+"""
+
+def download_model():
+    pass
+# download_model()
+
+
+"""
+Folder Structure - 
+./data/   
+    |-positive/
+        - xyz1.wav
+        - xyz2.wav
+        ...
+    |-negative/
+        - abc1.wav
+        - abc2.wav
+        ...
+    |-models/
+        -modelName.pkl
+"""
+
+def finetune(dataPath=data_folder, modelName="rnd1to10_stg4-rn50.pkl", newModelName="rnd1to10_stg4-rn50.pkl"):
+    """
+    Function to do finetuning of the model
+    """
+
+    # Definining Audio config needed to create on the fly mel spectograms
+    config = AudioConfig(
+        standardize=False,
+        sg_cfg=SpectrogramConfig(
+            f_min=0.0,  # Minimum frequency to Display
+            f_max=10000,  # Maximum Frequency to Display
+            hop_length=256,
+            n_fft=2560,  # Number of Samples for Fourier
+            n_mels=256,  # Mel bins
+            pad=0,
+            to_db_scale=True,  # Converting to DB sclae
+            top_db=100,  # Top decible sound
+            win_length=None,
+            n_mfcc=20,
+        ),
+    )
+    config.duration = 4000  # 4 sec padding or snip
+    config.resample_to = 20000  # Every sample at 20000 frequency
+    config.downmix = True
+
+    # Creating Data Loader
+    audios = (
+        AudioList.from_folder(data_folder, config=config)
```

Comment:
`dataPath` is the parameter, `data_folder` is just its default value.
```suggestion
        AudioList.from_folder(dataPath, config=config)
```